### ISSUE TITLE
Left navigation controls on Docs page do not have landmark role : A11y_Typespec_Keyboard.

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/typespec#10793]
**Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification.**
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."
### GitHub Tags:
#A11yTCS; A11yMAS; #A11ySev2; #DesktopWeb; #Win11; #Chrome; #E2E_TypeSpec_Web_May2026; #WCAG1.3.1; #TypeSpec; #Info and Relationship; #Element:Landmarks; #AIlimited;
### Issue Subcategory:
Important landmarks not defined. 
### User Experience:
Screen reader users cannot quickly jump to important regions such as the main content when those landmarks are missing, forcing more linear navigation.
### Repro Steps:
Launch url https://typespec.io/docs/> Get started button.
### Actual Result:
The left navigation controls on the Docs page are not defined with a proper landmark role (e.g., navigation). As a result, assistive technologies cannot identify this section as a navigation region.
### Expected Result:
Left navigation controls should be defined using a proper landmark role (e.g., <nav> or role="navigation") so they are accessible via landmark navigation.
### How to Fix:
Add the appropriate landmark for the missing region, such as <main> or role="main" for the primary content area. Use native landmarks where possible, or equivalent ARIA landmark roles when remediating existing markup. 
### Environment details:
Application Name: TypeSpec 
URL: [typespec.io](https://typespec.io/)
Edition Windows 11 Enterprise
Version 25H2 OS build 26200.8390
Chrome Version 148.0.7778.168 (Official Build) (64-bit)

### Attachment:

<img width="959" height="533" alt="Image" src="https://github.com/user-attachments/assets/e0ab1649-03a6-4ca5-8863-874806f061fd" />

**Comment by kupatkar99** [comment]
Verified the issue in below environment:
https://typespec.io/
Chrome: Version 150.0.7871.182 (Official Build) (64-bit)
issue is not fixed. Please find added attachment for reference.

<img width="1365" height="755" alt="Image" src="https://github.com/user-attachments/assets/0dad989a-b8ee-45f9-9066-d1a7e0f36c2b" />

**Comment by timotheeguerin** [comment]
Merging a workaround that "fixes" this but it seems like this is first an issue with the accessibility tool not showing the visual indication for landmark correctly because it is on a div which has no dimensions(still parent to the content)
